### PR TITLE
Edit Timeline Leap guides

### PR DESCRIPTION
**Pull request by ShinySylveon04**
# Guide Details

## I want to (select one)

- [ ] Add a guide
- [x] Modify an existing guide

## Games affected

- Sun/Moon/Ultra Sun/Ultra Moon

## Type of RNG

- Timeline Leap

## Language

- English

## Notes

This will close issue #123.

### CODE REVIEW

File: guides/Sun and Moon/Timeline Leap Guide.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Timeline Leap'
-description: "A timeline can predict Pokemon a player can obtain - Timeline leap allows a player to 'leap' onto specific timelines"
-slug: 'retail-sm-timeleap'
+description: "A timeline can predict Pokemon a player can obtain. Timeline leap allows a player to 'leap' onto specific timelines"
+slug: 'retail-usum-timeleap'
```
Comment (zaksabeast) [inline guides/Sun and Moon/Timeline Leap Guide.md:4]: Was this an intentional change to have one timeleap url?

If not, we may want to change it back:
```suggestion
slug: 'retail-sm-timeleap'
```

File: guides/Sun and Moon/Timeline Leap Guide.md [hunk 2]
```diff
@@ -57,22 +60,29 @@ You can use "Safe F Only" filter to find safe frames. You'll have to reset filte
 
 ## Step 6: Calculate the timeline leap frame
 
-Input the current frame you are on after confirming it is a safe frame into the starting frame in "Frame Range". Choose "Timeline Leap" and then "Calculate".
+1. Input the current frame you are on after confirming it is a safe frame into the starting frame in "Frame Range".
```
Comment (zaksabeast) [inline guides/Sun and Moon/Timeline Leap Guide.md:63]:
```suggestion
1. After confirming your current frame is a safe frame, input it into the "Frame Range" starting frame.
```

File: guides/Sun and Moon/Timeline Leap Guide.md [hunk 3]
```diff
@@ -57,22 +60,29 @@ You can use "Safe F Only" filter to find safe frames. You'll have to reset filte
...
+
+2. Then choose "Timeline Leap" and click on "Calculate".
```
Comment (zaksabeast) [inline guides/Sun and Moon/Timeline Leap Guide.md:65]:
```suggestion
2. Choose "Timeline Leap" and click on "Calculate"
```

File: guides/Sun and Moon/Timeline Leap Guide.md [hunk 4]
```diff
@@ -57,22 +60,29 @@ You can use "Safe F Only" filter to find safe frames. You'll have to reset filte
...
 
-- Within the window that popped up, choosing "Yes" will allow you to check that Frame 2 still has the spread you wanted.
-  - Choose "No" will allow you to check if there are any other Frame 1 that can make Frame 2 have the spread you wanted.
-- If Frame 2 does not have the spread you wanted, search for your target spread within the timeline created. (Choose "Yes" when the window pops up.) - If you did not mess with filters after finding your target frame, it will be easy to find.
-  - Right click and "Set as Target Frame" on the desired spread. This is now your new Frame 2.
-  - If Frame 2 is not on the timeline, then you'll have to find the new frame the spread is on. Choose "Frame Range" and see what frame the spread is now on and then redo the "Timeline Leap" to find the new Frame 1.
+1. Within the window that popped up, choosing "Yes" will allow you to check that Frame 2 still has the spread you wanted.
+
+   - Choose "No" will allow you to check if there are any other Frame 1 that can make Frame 2 have the spread you wanted.
```
Comment (zaksabeast) [inline guides/Sun and Moon/Timeline Leap Guide.md:78]:
```suggestion
   - Choosing "No" will allow you to check if there are any other Frame 1 that can make Frame 2 have the spread you wanted.
```

**Review by zaksabeast — CHANGES_REQUESTED** [review]
Sorry, I know you were just modifying formatting and my changes are more about the content.  Just a few small changes, but otherwise looks great.

I only suggested changes for the SM guide, but most changes should be reflects into the USUM guide.

File: guides/Sun and Moon/Timeline Leap Guide.md [hunk 5]
```diff
@@ -20,7 +20,10 @@ subCategory: 'Custom Firmware'
...
    - Do not check "Safe F Only".
 
-2. Search using "Frame Range" to find a frame you want to land on. Right click on the frame and "Set as Target Frame". This frame will be referred to as Frame 2 from here on.
+2. Search using "Frame Range" to find a frame you want to land on.
+
+3. Right click on the frame and "Set as Target Frame".
+   - This frame will be referred to as Frame 2 from here on.
    - Any frame that shows up can be landed on with this method, so if the frame isn't a safe frame you can still use it as a target frame.
 
 ## Step 2: Advance frames
```
Comment (RichardPaulAstley) [inline guides/Sun and Moon/Timeline Leap Guide.md:29]: I don't see any comment about when you should leave the Festival Plaza / Mall to start the timeleap process

File: guides/Sun and Moon/Timeline Leap Guide.md [hunk 6]
```diff
@@ -84,14 +94,17 @@ Due to the inaccurate assumption of 3DSRNGTool "Frame Range" mode, Frame 2 may o
...
+2. Closing the X menu
 
-2. Closing the X menu - The game will use 2 RNG calls for the next timing of the character fidgets.
+   - The game will use 2 RNG calls for the next timing of the character fidgets.
 
-3. Press `A` at YES/NO screen (Mystery Gifts) - The game will use around 10 RNG calls to generate 1 Pokemon for testing.
+3. Press `A` at YES/NO screen (Mystery Gifts)
+   - The game will use around 10 RNG calls to generate 1 Pokemon for testing.
 
 Since you can jump between different timelines by choosing the leap timing, you can land on any frame of your choosing.
```
Comment (RichardPaulAstley) [inline guides/Sun and Moon/Timeline Leap Guide.md:110]: Also a way to say that it's not doable on wild stuff to be 100% complete ?

File: guides/Ultra Sun and Ultra Moon/Timeline Leap Guide.md [hunk 7]
```diff
@@ -20,7 +20,10 @@ subCategory: 'Custom Firmware'
...
    - Do not check "Safe F Only".
 
-2. Search using "Frame Range" to find a frame you want to land on. Right click on the frame and "Set as Target Frame". This frame will be referred to as Frame 2 from here on.
+2. Search using "Frame Range" to find a frame you want to land on.
+
+3. Right click on the frame and "Set as Target Frame".
+   - This frame will be referred to as Frame 2 from here on.
    - Any frame that shows up can be landed on with this method, so if the frame isn't a safe frame you can still use it as a target frame.
 
 ## Step 2: Advance frames
```
Comment (RichardPaulAstley) [inline guides/Ultra Sun and Ultra Moon/Timeline Leap Guide.md:29]: Same as USUM

File: guides/Ultra Sun and Ultra Moon/Timeline Leap Guide.md [hunk 8]
```diff
@@ -84,14 +94,17 @@ Due to the inaccurate assumption of 3DSRNGTool "Frame Range" mode, Frame 2 may o
...
+2. Closing the X menu
 
-2. Closing the X menu - The game will use 2 RNG calls for the next timing of the character fidgets.
+   - The game will use 2 RNG calls for the next timing of the character fidgets.
 
-3. Press `A` at YES/NO screen (Mystery Gifts) - The game will use around 10 RNG calls to generate 1 Pokemon for testing.
+3. Press `A` at YES/NO screen (Mystery Gifts)
+   - The game will use around 10 RNG calls to generate 1 Pokemon for testing.
 
 Since you can jump between different timelines by choosing the leap timing, you can land on any frame of your choosing.
```
Comment (RichardPaulAstley) [inline guides/Ultra Sun and Ultra Moon/Timeline Leap Guide.md:110]: Maybe a note for some cases that are really unstable like Poipole. 
Also a way to say that it's not doable on wild stuff to be 100% complete ?

**Review by RichardPaulAstley — CHANGES_REQUESTED** [review]

**Review by RichardPaulAstley — APPROVED** [review]

**Pull request closed without merging**